### ISSUE TITLE
Huge saturnus + trisagion issue

### ISSUE THREAD

**Issue by Lonrite** [UnderscoreKilburn/repentance-issues#1137]
Saturnus pushes away your bombs with Trisagion, which is entirely inconsistent with how base tears work with Saturnus (they don't push your bombs)

this feels really devastating especially in a daily context, in today's daily I got hit for 2 hits with my own ricocheted bomb at myself because of it, as well as wasting 5-10 bombs trying to blow up doors and make rush, only for Trisagion to constantly push them away

this is really annoying mostly because of how inconsistent it is, sorry if I sound snarky this synergy was just an ass. hope this gets fixed 1 day lol

**Comment by Turtier** [comment]
Not exactly the same thing but tiny planet has a similar issue. Saturnus trisagion def should be fixed to not push bombs and then for tiny planet, trisagion shots in general should just consistently disappear after 20-30 seconds or something which would also prevent #934
https://www.twitch.tv/turtier/clip/SuspiciousBloodyRadishWholeWheat-B8wwPdQGH6BHTr2F

**Comment by Lonrite** [comment]
yeah it's extremely annoying, I also wasted tons of time today up to 5+ minutes because of being forced waiting for it to lift off, so I can actually find secret rooms for once.
